DabraHalleloEndaAbunaSamuel/DHAS008.xml

```diff
@@ -0,0 +1,254 @@
+                                        <height>230</height>
+                                        <width>190</width>
+                                        <depth>100</depth>
+                                  
+                                   
+                                    </dimensions>
+                                    
+                                </extent>                                
+                                <condition key="deficient">Both wooden boards are broken, crudely repaired. The manuscript is 
+                                    crudely resewn, the sewing is damaged.</condition>
+                            </supportDesc>
+                            
+                        </objectDesc>
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>
+                                <decoNote xml:id="b2" type="Cover">Remains of the leather cover.</decoNote>
+                                <decoNote xml:id="b3" type="SewingStations">4</decoNote>
+                               <decoNote xml:id="b4" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>                                    
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                        </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1849" source="lettering">17th/early 19th century</origDate>.
+                            The manuscript is composed of two parts, the more recent <ref target="#1">Unit 1</ref> and the earlier <ref target="#2">Unit 2</ref>.
+                        </origin>
+                    </history>
+                        <additional>
+                            
+                            <adminInfo>
+                                <recordHist>
+                                    <source/>                                  
+                                          
+                                </recordHist>
+                                
+                            </adminInfo>
+                        </additional>
+                        
+                         
+                         <msPart xml:id="p1">
+                             <msIdentifier>
+                                 <idno/>
+                             </msIdentifier>
+                             <msContents>
+                                 <summary/>            
+                         <msItem xml:id="p1_i1">
+                             <locus from="1ra" to="185vb"/>
+                             <locus target="#196r"/>
```

Comment:
Are you sure the locus target is 196r? It looks confusing, but you might be right.

Reply:
Yes. Unit 2 is inserted just before the last page of Uni 1.

Reply:
But you are right, smth is wrong, I am correcting